examples/tabs/js/alt.js

```diff
@@ -0,0 +1,4 @@
+var Alt = require('../../../src/alt')
+var alt = new Alt()
+
+module.exports = alt
```

Comment:
Yeah, definitely +1 to consistency. All the stuff in guide/ and docs/ has semicolons and the inline examples in mixins and utils have semicolons as well.